src/gameboy/PPU/PPU.cpp

```diff
@@ -359,9 +355,9 @@ bool PPU::meetsStatConditions() const {
 
 void PPU::dma_transfer(uint8_t startAddress) {
     if (0x00 <= startAddress && startAddress <= 0xdf) {
-        uint16_t start_addr = (startAddress << 8);
+        uint16_t startAddr = (startAddress << 8);
         for (uint8_t i = 0; i <= 0x9f; i++) {
-            this->memory->write(OAM_START + i, this->memory->read(start_addr+i));
+            this->memory->write(OAM_START + i, this->memory->read(startAddr + i));
```

Comment:
this

Reply:
Removed more "this->" than commented, please re-review.